integration/test_events_test.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/GoogleContainerTools/skaffold/integration/skaffold"
+	"github.com/GoogleContainerTools/skaffold/proto/v1"
+)
+
+func TestTestEvents(t *testing.T) {
+	MarkIntegrationTest(t, CanRunWithoutGcp)
+
+	tests := []struct {
+		description  string
+		podName      string
+		expectedText string
+		testDir      string
+		testFile     string
+		config       string
+		testType     string
+	}{
+		{
+			description:  "test events for custom test",
+			podName:      "custom-test-events",
+			expectedText: "bar\n",
+			testDir:      "testdata/test-events/custom",
+			testFile:     "testdata/test-events/custom/test",
+			config:       "skaffold.yaml",
+			testType:     "custom",
+		},
+		{
+			description: "test events for structure test",
+			podName:     "structure-test-events",
+			testDir:     "testdata/test-events/structure",
+			testFile:    "testdata/test-events/test",
+			config:      "skaffold.yaml",
+			testType:    "structure",
+		},
```

Comment:
have a case that runs both. See below comment about using `profiles`

Reply:
Updated test case.